### ISSUE TITLE
Enable loop even on items.lenth < slidesPerView * 2

### ISSUE THREAD

**Issue by rbalet** [nolimits4web/swiper#6600]
### Clear and concise description of the problem

Actually, when the total number of slides is < slidesPerView * 2, the swiper container will show blank slides after it crashed.  

Because it cannot move the slides the way it would have liked to

### Suggested solution

We could, if we have to few slides, just stay at the last slide, so ignoring the loop entirely. 

Or slide a the beginning back again, in case of AutoPlay.

### Alternative

**Duplicate slides**  
Doing as before, meaning if we do not have enough slides, replicating the slides to get exactly slidesPerView * 2

**Limit the slidesPerView**   
The slides per view could also be updated based on the number of slides

### Additional context

For what I saw over on the internet, this break a lot of application.

The first proposed suggestion is what I found  over on stack overflow 

* [Slide back to the beginning](https://stackoverflow.com/a/73143751/11135174)
* [Update bas on the number of slides](https://stackoverflow.com/a/75617304/11135174)

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### Would you like to open a PR for this feature?

- [ ] I'm willing to open a PR

**Comment by smerriman** [comment]
I agree this needs fixing. I disagree with a couple of your suggested approaches.

Consider this from a client's perspective, who wants a carousel with 6 visible items, which is set to loop like every other carousel they've seen online (and remembering the origin of the word carousel; something circular that rotates).

They add 10 items, swipe, and the carousel breaks entirely. 

Telling them this is intended by the developers makes zero sense to them; they want it to work.

Changing the carousel to not loop, or to display a different number of items, equally aren't valid responses.

The only option actually involves it loopping properly like it always used to / like requested.

**Comment by NathanBeddoeWebDev** [comment]
Just to try and fill this issue out a bit more.

https://codesandbox.io/p/sandbox/swiper-infinite-loop-forked-nrs9fc?file=%2Findex.html is an example of where this happens. It works in version 8, but it is a massive blocker in version 9.

The carousel just completely swaps, I'm guessing the issue is somewhere in loopFix.js where the next slides aren't being appended properly to that list.

**Comment by milbau** [comment]
I'm also looking forward to fixing this issue because I'm having trouble with this in several projects.
